db/seeds.rb

```diff
@@ -1,7 +1,74 @@
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+photo1 = 'https://images.unsplash.com/photo-1624574337423-7ea725c5540c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=869&q=80'
+photo2 = 'https://images.unsplash.com/photo-1489980557514-251d61e3eeb6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80'
+photo3 = 'https://images.unsplash.com/photo-1524660988542-c440de9c0fde?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80'
+photo4 = 'https://images.unsplash.com/photo-1487573884658-a5d3c667584e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=803&q=80'
+user1 = User.create!(name: 'Francis Ugorji', photo: photo1, bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed neque enim, viverra nec quam sit amet, suscipit aliquam nulla. - Francis Ugorji', posts_counter: 0)
+user2 = User.create!(name: 'Brian Orora', photo: photo2, bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed neque enim, viverra nec quam sit amet, suscipit aliquam nulla. - Brian Orora', posts_counter: 0)
+user3 = User.create!(name: 'Toluwase Ajise', photo: photo3, bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed neque enim, viverra nec quam sit amet, suscipit aliquam nulla. - Toluwase Ajise', posts_counter: 0)
+user4 = User.create!(name: 'Chaw Nguni', photo: photo4, bio: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed neque enim, viverra nec quam sit amet, suscipit aliquam nulla. - Chaw Nguni', posts_counter: 0)
+
+lorem_donec = 'Donec dapibus consequat tellus a pulvinar. Fusce auctor rhoncus orci a semper. Sed interdum accumsan ullamcorper.'
+lorem_mauris = 'Mauris hendrerit odio eu quam dapibus, sit amet malesuada quam pulvinar. Ut quis venenatis ex, eu tristique orci.'
+
+post1 = user1.posts.create!(title: 'Technology', text: lorem_donec, likes_counter: 0, comments_counter: 0)
+post2 = user1.posts.create!(title: 'Science', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
+
+post3 = user2.posts.create!(title: 'Business', text: lorem_donec, likes_counter: 0, comments_counter: 0)
+post4 = user2.posts.create!(title: 'Education', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
+
+post5 = user3.posts.create!(title: 'Sports', text: lorem_donec, likes_counter: 0, comments_counter: 0)
+post6 = user3.posts.create!(title: 'Agriculture', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
+
+post7 = user4.posts.create!(title: 'Arts', text: lorem_donec, likes_counter: 0, comments_counter: 0)
+post8 = user4.posts.create!(title: 'Infrastructure', text: lorem_mauris, likes_counter: 0, comments_counter: 0)
```

Comment:
- [ ] Please fix the `likes_counter` typo here in the seeds and across your project, check the column name in your schema

Reply:
Hi @AshakaE 👋, 

I am sorry but I think you got it wrong. I used `likes_counter` as a column for my database table. If you check the column table for posts in the `schema.rb`,  you will see this:

![image](https://user-images.githubusercontent.com/77106962/150210212-8e51ad21-1746-4a16-a907-febd2b7e18e7.png)

The `likes_counter` functions as you can see in the image below:
![image](https://user-images.githubusercontent.com/77106962/150210044-cfd5728c-d9b6-4d4d-b950-fc72b36d9f8b.png)

Finally, if the `likes_counter` was not functioning, the `create!` method will throw an error when creating the posts. 😃 👍

Reply:
- Everything looks fine to me.